### PR TITLE
Changed us Host in QUICK-START.md

### PR DESCRIPTION
**Pull request by it-hms**
This commit changes the host parameter based on feedback from Software AG about the public instance URLs. The path us-latest.comulocity.com is not used.

### CODE REVIEW

File: QUICK-START.md [hunk 1]
```diff
@@ -83,21 +83,22 @@ release package.
...
 
 ![Connector Configuration File Bootstrap Edit](images/ConnectorConfigBootstrap.png)
 
-Typically, the following example values are the standard values for all public Cumulocity instances
-maintained by SoftwareAG on its **EU** or **US** servers,
+The following example values are the standard values for all public Cumulocity instances maintained by 
+SoftwareAG on its **EU** or **US** servers. Dedicated and "Experimental" instances may have different values.
 
-The `BootstrapTenant` is the bootstrap tenantID which is `management`;
+The `BootstrapTenant` is the bootstrap tenantID, which is `management`
```
Comment (alexjhawk) [inline QUICK-START.md:93]:
```suggestion
The `BootstrapTenant` is the bootstrap tenantID, which is `management`.
```

File: QUICK-START.md [hunk 2]
```diff
@@ -83,21 +83,22 @@ release package.
...
 
-The `BootstrapUsername` is the bootstrap username, which is `devicebootstrap`;
+The `BootstrapUsername` is the bootstrap username, which is `devicebootstrap`
```
Comment (alexjhawk) [inline QUICK-START.md:95]:
```suggestion
The `BootstrapUsername` is the bootstrap username, which is `devicebootstrap`.
```

File: QUICK-START.md [hunk 3]
```diff
@@ -83,21 +83,22 @@ release package.
...
 
-The `Host` for the EU server is `mqtt.eu-latest.cumulocity.com`;
-the `Host` for the US server is `mqtt.us-latest.cumulocity.com`;
+The `Host` for the EU server is `mqtt.eu-latest.cumulocity.com`
 
-And the `BootstrapPassword` is `Fhdt1bb1f`.
+The `Host` for the US server is `mqtt.us.cumulocity.com`
```
Comment (alexjhawk) [inline QUICK-START.md:99]:
```suggestion
The `Host` for the US server is `mqtt.us.cumulocity.com`.
```

File: QUICK-START.md [hunk 4]
```diff
@@ -83,21 +83,22 @@ release package.
...
+
+And the `BootstrapPassword` is `Fhdt1bb1f`
```
Comment (alexjhawk) [inline QUICK-START.md:101]:
```suggestion
The `BootstrapPassword` for EU and US servers is `Fhdt1bb1f`.
```

**Review by alexjhawk — CHANGES_REQUESTED** [review]
The content looks good, although there are a couple of minor suggestions:

- Semicolons should become periods instead of being eliminated (to maintain proper punctuation.)
- The `BootstrapPassword` line should be updated to become a standalone sentence, especially after removing the semicolon(s) above.

**Comment by alexjhawk** [comment]
@it-hms Were there any changes between the last review and now? It looks unchanged.

**Comment by it-hms** [comment]
> @it-hms Were there any changes between the last review and now? It looks unchanged.

missed the git add

**Review by alexjhawk — APPROVED** [review]

**Pull request merged**